src/test/java/com/epam/tc/hw2/Utils.java

```diff
@@ -0,0 +1,38 @@
+package com.epam.tc.hw2;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Utils {
+
+    public static final String PAGE_URL = "https://jdi-testing.github.io/jdi-light/index.html";
+    public static final String HOME_PAGE_TITLE = "Home Page";
+    public static final String LOGIN = "Roman";
+    public static final String PASSWORD = "Jdi1234";
+    public static final String USERNAME = "ROMAN IOVLEV";
+    public static final List<String> headerItems = List.of(
+            "HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"
+    );
+    public static final int IMAGES_NUMBER = 4;
+    public static final List<String> iconsText = List.of(
+            "To include good practices\nand ideas from successful\nEPAM project",
+            "To be flexible and\ncustomizable",
+            "To be multiplatform",
+            "Already have good base\n(about 20 internal and\nsome external projects),\nwish to get more…"
+    );
+    public static final String FRAME_BUTTON = "Frame Button";
```

Comment:
What is the reason of placing such constants? It used in only one place.

Reply:
Such constants are converted to just static variables